entrypoint.ps1

```diff
@@ -106,10 +115,6 @@ function Initialization {
             exit 1
         }
     }
-
-    end {
```

Comment:
Could we this block in, we'd like to have a consistent code structure across all functions?

Reply:
Can do that, sure.
Comes at a slight performance cost, so I was instead planning in purging it from all commands on the refactor (can do that in bulk then though, so undoing that)